ui/css/treeherder-info-panel.css

```diff
@@ -486,14 +503,90 @@ div.performance-panel {
   height: 14px;
   overflow: hidden;
   padding-left: 5px;
-  color: darkgray;
+  font-family: Menlo,Monaco,Consolas,"Courier New",monospace;
 }
 
 .failure-line-message-expanded {
   margin-left: 5px;
   margin-bottom: 10px;
-  color: #747474;
   overflow-x: auto;
   white-space: pre-wrap;
   font-family: Menlo,Monaco,Consolas,"Courier New",monospace;
 }
+
+.classification-line {
+  display: flex;
+  margin-bottom: 6px;
+}
+
+.classification-line-actions {
+  width: 70px;
+  text-align: center;
+}
+
+.classification-line-actions > span > select {
+  width: 60px;
+  margin-bottom: 6px;
+}
+
+.classification-line-detail {
+  width: 100%;
+  padding-top: 3px;
+  padding-bottom:3px;
+  border-bottom: thin solid #eee;
+}
+
+.classification-set-bug {
+  cursor: pointer;
+  font-size: 8px;
+}
+
+.classification-icon {
+  width: 24px;
+  margin-top: auto;
+  margin-bottom: auto;
+  text-align: center;
+  display: inline-block;
+}
+
+.classification-icon > * {
+ margin-left: auto;
+ margin-right: auto;
+}
+
+/* TODO: Make this better */
```

Comment:
Well...  why aren't you making this better?  Why?  Do you hate when things look fabulous?  Hmm..  perhaps darkgrey is already fabulous.  wfm  :)